Homework1/Program.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Homework1
+{
+    internal static class Program
+    {
+        private static readonly List<Person> Persons = new List<Person>();
+
+        public static void Main(string[] args)
+        {
+            bool keepGoing = true;
+            do
+            {
+                Person person_ = GettingInfos();
+                if (person_ == null)
+                    break;
+                Console.WriteLine("Do you want to show all persons ? ( y/n )");
+                string resp = Console.ReadLine();
+                if (resp == "yes" || resp == "y")
+                {
+                    foreach (var person in Persons)
+                    {
+                        Console.WriteLine($"{person.Name} is {person.Age} years old, his/her weight is {person.Weight} kg and his/her height is {person.Height} cm.");
+                        Console.WriteLine($"Their BMI is {(decimal)person.Weight / (decimal)(person.Height * person.Height)}\n");
+                    }
+                }
+
+                Console.WriteLine("Do you want to continue? ( y/n )");
+                resp = Console.ReadLine();
+                if (resp == "no" || resp == "n")
+                {
+                    keepGoing = false;
+                }
+            } while (keepGoing);
```

Comment:
Too much is going on. Please split into functions.

Reply:
Thanks for the review I will finish when I'll have time

Reply:
Made void AskShowingAllPersons() & bool AskContinue()